src/components/views/elements/MemberEventListSummary.js

```diff
@@ -379,7 +380,7 @@ module.exports = React.createClass({
 
         if (fewEvents) {
             return (
-                <div className="mx_MemberEventListSummary">
+                <div className="mx_MemberEventListSummary" data-contained-scroll-tokens={eventIds}>
```

Comment:
what does this actually expand to in the HTML for multiple eventIds? I suspect it's `<div data-contained-scroll-tokens="event1" data-contained-scroll-tokens="event2" data-contained-scroll-tokens="event3">`, which, as you can see, gets wordy quickly.